.github/workflows/build_macOS.yaml

```diff
@@ -7,6 +7,7 @@ jobs:
       VERBOSE: 1
     runs-on: macos-10.15
     strategy:
+      fail-fast: false
```

Comment:
you want this to continue even if one of the jobs fail?

Reply:
I was using this for testing, but I like it. If the MPICH build fails why should OpenMPI not continue?